### ISSUE TITLE
[Switch, Select, List]: [Accessibility issues]

### ISSUE THREAD

**Issue by jinnatan** [SAP/ui5-webcomponents-react#6349]
### Describe the bug

Getting accessibility issue when testing with Tool Access Continuum for Switch, List(ListItemStandard), Select, SegmentedButton(SegmentedButtonItem). 

Select, SegmentedButton(SegmentedButtonItem): 
```
Accessibility Concern: This DIV (role=combobox) must have all of the following attributes: aria-controls | aria-expanded

But if I tried to add the attribute: e.g aria-expanded, i got another error: 
Accessibility Concern: The aria-expanded attribute is not allowed on this UI5-SELECT
```
  
 Switch (aria-required  attribute is not added as property of Switch)
 ```
Accessibility Concern: The aria-required attribute is not allowed on this DIV
```

List(ListItemStandard)
```
Accessibility Concern: This LI does not have a ul element (without an ARIA-assigned role); ol element (without an ARIA-assigned role); an element with a role set to the value: list as a parent; or a ul element (without an ARIA-assigned role), ol element (without an ARIA-assigned role) or element with a role set to the value 'list' with an aria-owns attribute set to the ID of the element in the same DOM
```

### Reproduction steps

Just use the controls, the accessibility test failed. 
This happens in "@ui5/webcomponents-react": "1.29.4"  and  "@ui5/webcomponents-react": "2.1.0"

```
 <Switch onChange={function _s() { }} />

        <Select
          valueState="None"
          accessibleName="Select"
        >
          <Option>
            Option 1
          </Option>
        </Select>


        
        <List
          growing="None"
          headerText="List with ListItemStandard"
          selectionMode="None"
          separators="All"
        >
          <ListItemStandard additionalText="3" >
            List Item 1
          </ListItemStandard>
        </List>
```
        

### UI5 Web Components for React Version

2.1.0

### UI5 Web Components Version

2.2.0

### Browser

Chrome

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by MarcusNotheis** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.
